source/_integrations/google_assistant.markdown

```diff
@@ -109,6 +109,12 @@ If you want to support active reporting of state to Google's server (configurati
 
 ### Enable Local Fulfillment
 
+<div class='note'>
+
+Local fulfillment will be disabled if Home Assistant is configured to use HTTPS.
+
+</div>
```

Comment:
Does this have to be a big note box?

This can be described as a normal introduction paragraph with the requirements for local fulfillment as well, right?

Reply:
I looked at other note boxes and thought this information to be similar.

It is an important "gotcha" that being clearly communicated could solve users a lot of headaches. I only found myself while searching through the source code and found the code that disabled local fulfillment.

You could easily be correct that this isn't the best way to present this information, however maybe it is good enough until someone more qualified than myself writes an introduction paragraph for local fulfillment.

Reply:
I think we should avoid having these boxes everywhere.
Please make the adjustment as requested.